src/pages/AddPersonalBankAccountPage.js

```diff
@@ -21,6 +22,8 @@ const AddPersonalBankAccountPage = props => (
                 BankAccounts.addPersonalBankAccount(account, password, plaidLinkToken);
             }}
             onExitPlaid={Navigation.dismissModal}
+            receivedRedirectURI={getPlaidOAuthReceivedRedirectURI()}
+            plaidLinkOAuthToken={props.plaidLinkToken}
```

Comment:
Where does this come from?

Reply:
Good catch, think I originally had a different idea for this page but now we need it to be a HOC connected to the `plaidLinkToken` onyx key.